toolchain/check/context.cpp

```diff
@@ -373,14 +373,14 @@ class TypeCompleter {
       return true;
     }
 
-    auto node_id = context_.sem_ir().GetTypeAllowBuiltinTypes(type_id);
-    auto node = context_.nodes().Get(node_id);
+    SemIR::InstId inst_id = context_.sem_ir().GetTypeAllowBuiltinTypes(type_id);
```

Comment:
Preserve `auto` for now:
```suggestion
    auto inst_id = context_.sem_ir().GetTypeAllowBuiltinTypes(type_id);
```